app/components/add-layer/template.hbs

```diff
@@ -1,19 +1,25 @@
 {{#liquid-if isOpen use='toDown'}}
     <div class="add-layer {{if isOpen 'open'}}">
         <div class="container">
+          
             <div class="row p-v-md">
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Wiki</button>
+                
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Add Wiki layer </button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Link</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Add Link layer</button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'file'}}>{{fa-icon "download"}} <br /> Download</button>
+                <div class="col-xs-2">
```

Comment:
XS-2 will make it tiny in phone screens. Check it out with dev tools but I think it should be sm-2